model_zoo/tests/model_mnist_distributed_io.prototext

```diff
@@ -80,6 +80,7 @@ model {
   layer {
     name: "1"
     parents: "1"
+    children: ""
```

Comment:
We can remove this field here and everywhere else it appears in the pull request.